[patch]: bss_file.c: don't force largefile mode

[Adapted to version 1.0.2]

Signed-off-by: Vicente Olivert Riera <Vincent.Riera@imgtec.com>

diff -rup a/crypto/bio/bss_file.c b/crypto/bio/bss_file.c
--- a/crypto/bio/bss_file.c	2015-01-22 14:58:32.000000000 +0000
+++ b/crypto/bio/bss_file.c	2015-01-26 16:27:52.972519876 +0000
@@ -78,9 +78,6 @@
  * of 32-bit platforms which allow for sequential access of large files
  * without extra "magic" comprise *BSD, Darwin, IRIX...
  */
-#  ifndef _FILE_OFFSET_BITS
-#   define _FILE_OFFSET_BITS 64
-#  endif
 # endif
 
 # include <stdio.h>
